app/routes/manifest[.]webmanifest.ts

```diff
@@ -0,0 +1,75 @@
+import { json } from "@remix-run/node";
+
+export const loader = async () =>
+  json(
+    {
+      short_name: "Shelf",
+      name: "Shelf | Asset Management Made Simple",
+      start_url: "/",
+      display: "standalone",
+      background_color: "#d3d7dd",
+      theme_color: "#c34138",
+      shortcuts: [
+        {
+          name: "Homepage",
+          url: "/",
+          icons: [
+            {
+              src: "/icons/android-icon-96x96.png",
```

Comment:
Those are just default remix icons. All of them need to be replaced by shelf icons. Please ask @jurrejansen for help if needed. Maybe you can even use something like this to save time and make it easier: https://tools.crawlink.com/tools/pwa-icon-generator/